### DISCUSSION TITLE
It would be better if a "mark malt outline" function is added

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#112] [Feedback & Feature Requests]
Is is possible to add a "mark malt outline" function to the right click when an edge is selected?
I think this would be very helpful to add outline to specific edge.

Maybe it is already possible in the current mode.

**Comment by pragma37** [comment]
Adding custom edge data is not doable from Blender addons.
What would be possible is to use the "Mark Freestyle Edge" on Malt too.

Other than that, I agree this is a very desirable feature.
I made a prototype for this but needs extra work.
